superset/tags/models.py

```diff
@@ -116,13 +119,15 @@ class ObjectUpdater:
 
     @classmethod
     def get_owners_ids(
-        cls, target: Union["Dashboard", "FavStar", "Slice"]
+        cls, target: Union["Dashboard", "FavStar", "Slice", "Query", "SqlaTable"]
```

Comment:
Let's get rid of these quotation marks by doing the ol' `from __future__ import annotations